custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -81,6 +96,40 @@ def _apply_sklearn_transformer(data, model):
             ) from None
         return output_data
 
+    def _load_custom_hooks_for_new_drum(self, custom_module):
```

Comment:
load class based and module based hooks.

Reply:
i like that better, but mind if we do it in a follow-up so we don't add merge conflicts to @drenfr01 and I's subsequent PRs